copier/copier.go

```diff
@@ -200,6 +200,8 @@ type StatForItem struct {
 	IsRegular       bool   // dereferenced value for symlinks
 	IsArchive       bool   // dereferenced value for symlinks
 	ImmediateTarget string `json:",omitempty"` // raw link content
+	Uid             uint32
```

Comment:
I'm surprised by this, but apparently lint wants this to be `UID`?

Reply:
Hmm, let me double check - seems that this is not required at all.